### PR TITLE
Migrate to @dojo scoped package(s)

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue

**Description:**

Update `package.json` and all dependency references to use `@dojo` namespace.

Related to https://github.com/dojo/meta/issues/129

### CODE REVIEW

**Comment by agubler** [comment]
This will need some additional changes to support command discovery using the `@dojo` scope

File: package.json [hunk 1]
```diff
@@ -37,8 +37,8 @@
   "devDependencies": {
     "@types/sinon-as-promised": "^4.0.5",
     "codecov.io": "0.1.6",
-    "dojo-interfaces": "^2.0.0-alpha.6",
-    "dojo-loader": ">=2.0.0-beta.7",
+    "@dojo/interfaces": "2.0.0-alpha.11",
+    "@dojo/loader": "2.0.0-beta.7",
```
Comment (dylans) [inline package.json]: Probably want beta.8 here?

Comment (agubler) [inline package.json]: Yup I am re updating these as I go along.

**Comment by rishson** [comment]
👍

**Comment by agubler** [comment]
@dylans updated the package.json to the correct loader version.

**Pull request merged**